woocommerce/woocommerce-gateway-stripe
# Add SEPA UPE payment method

# Changes proposed in this Pull Request:

Issue #1771 

This PR integrates SEPA into UPE by creating a new UPE class for SEPA and adding it to the available UPE payment methods.


# Testing instructions

1. Ensure you have `_wcstripe_feature_upe` set to `yes` in the database and SEPA enabled in UPE – you might need to use the legacy settings interface to accomplish that.
2. In order to have the order updated correctly, make sure you run `stripe listen --forward-to 'http://localhost:8082/?wc-api=wc_stripe'` and set the webhook secret in the settings.
3. Visit the store and add something to the cart.
4. Go to checkout and you should see the new SEPA interface (screenshot below).
5. It should also work fine when paying with/saving it.

<img src="https://user-images.githubusercontent.com/10233985/131924712-290b1cc3-c8de-4cc5-8a86-3b82d815fd81.png" alt="screenshot" width="400"/>

-------------------
